torchrl/envs/common.py

```diff
@@ -2421,6 +2421,10 @@ def _rollout_stop_early(
                     tensordict = tensordict.to(policy_device, non_blocking=True)
                 else:
                     tensordict.clear_device_()
+
```

Comment:
This is not ideal and just a current solution, Id be interested to hear what you think @vmoens. We need to detach those values similar to [this](https://github.com/danijar/dreamer/blob/master/dreamer.py#L276) maybe via a detach transform?

Reply:
do we need to backprop anything in this loop?

Reply:
I added a transform to the mb env,
 can you confirm that we always want to detach these values in `mbenv.step`?